.github/workflows/package.yml

```diff
@@ -50,42 +62,74 @@ jobs:
+          pack build "$GHCR_IMAGE_LOCATION" $LATEST_TAG \
             --builder paketobuildpacks/builder-jammy-full \
             --publish
 
-          echo "ghcr_image_location=$GHCR_IMAGE_LOCATION" >> $GITHUB_OUTPUT
+      - name: Build and publish to ECR
+        id: build_and_publish
+        run: |
+          AWS_ACCOUNT_ID=${{ secrets.CI_AWS_ACCOUNT}}
+          AWS_REGION="eu-west-2"
+          REPO_NAME="funding-service-ci-shared-app"
+          IMAGE_TAG=${{ github.sha }}
+
+          ECR_IMAGE_URI="$AWS_ACCOUNT_ID.dkr.ecr.$AWS_REGION.amazonaws.com/$REPO_NAME:$IMAGE_TAG"
+          echo "::add-mask::$ECR_IMAGE_URI"
 
+          LATEST_TAG=""
+          if [ "${{ github.ref_name }}" == "main" ]; then
+            LATEST_TAG="--tag $AWS_ACCOUNT_ID.dkr.ecr.$AWS_REGION.amazonaws.com/$REPO_NAME:latest"
+          fi
+
+          uv export --format requirements-txt --no-hashes > requirements.txt
+
+          pack build "$ECR_IMAGE_URI" $LATEST_TAG \
+            --builder paketobuildpacks/builder-jammy-full \
+            --publish
+
+      - name: Export image URIs
+        id: export_uris
+        run: |
+          GHCR_IMAGE_URI="ghcr.io/${{ github.repository }}:${{ github.sha }}"
+          ECR_IMAGE_URI="${{ secrets.CI_AWS_ACCOUNT }}.dkr.ecr.eu-west-2.amazonaws.com/funding-service-ci-shared-app:${{ github.sha }}"
+      
+          echo "::add-mask::$ECR_IMAGE_URI"
+          echo "ghcr_image_location=$GHCR_IMAGE_URI" >> $GITHUB_OUTPUT
+          echo "ecr_image_location=$ECR_IMAGE_URI" >> $GITHUB_OUTPUT
+          
   notify_slack:
-    needs:
-      - paketo_build
-    if: ${{ always() && needs.paketo_build.result == 'failure' && github.ref_name == 'main'}}
-    uses: communitiesuk/funding-service-design-workflows/.github/workflows/notify-slack-deployment-failed.yml@main
-    with:
-      app_name: FS Paketo
-      env_name: 'build'
-      github_username: ${{ github.actor }}
-      workflow_url: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
-      compare_url: ${{ github.event_name == 'push' && github.event.compare || null }}
-    secrets:
-      SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
-      SLACK_NOTIFICATION_CHANNEL_ID: ${{ secrets.TEMP_SLACK_CHANNEL_ID }}
+      needs:
+        - paketo_build
+      if: ${{ always() && needs.paketo_build.result == 'failure' && github.ref_name == 'main'}}
+      uses: communitiesuk/funding-service-design-workflows/.github/workflows/notify-slack-deployment-failed.yml@main
+      with:
+        app_name: FS Paketo
+        env_name: 'build'
+        github_username: ${{ github.actor }}
+        workflow_url: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
+        compare_url: ${{ github.event_name == 'push' && github.event.compare || null }}
+      secrets:
+        SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+        SLACK_NOTIFICATION_CHANNEL_ID: ${{ secrets.TEMP_SLACK_CHANNEL_ID }}
```

Comment:
Please can you install the pre-commit hooks 🙏 

`pre-commit install --install-hooks` in the root of the repo